modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -72,17 +76,29 @@
 public class ObjectRelationshipLocalServiceImpl
 	extends ObjectRelationshipLocalServiceBaseImpl {
 
-	@Indexable(type = IndexableType.REINDEX)
 	@Override
 	public ObjectRelationship addObjectRelationship(
 			long userId, long objectDefinitionId1, long objectDefinitionId2,
 			String deletionType, Map<Locale, String> labelMap, String name,
 			String type)
 		throws PortalException {
 
+		return addObjectRelationship(
+			userId, objectDefinitionId1, objectDefinitionId2, deletionType,
+			labelMap, name, type, null);
+	}
+
+	@Indexable(type = IndexableType.REINDEX)
+	@Override
+	public ObjectRelationship addObjectRelationship(
+			long userId, long objectDefinitionId1, long objectDefinitionId2,
+			String deletionType, Map<Locale, String> labelMap, String name,
+			String type, Long parameterObjectFieldId)
```

Comment:
I'm still confused about the `parameterObjectFieldId` coming after the `type`.

Reply:
As discussed offline, I'll follow previous precedent.